### DISCUSSION TITLE
How to get rid `/dev/null` directory on leftover older version of MSYS2?

### DISCUSSION THREAD

**Discussion by gphg** [ScoopInstaller/Main#4427] [Q&A] [answered]
I have this on `scoop\apps\msys2`:
![image](https://user-images.githubusercontent.com/46024870/217187061-f1eecab9-fc37-493d-b866-53f19d5ab445.png)
There are 2022 directory (old; not linked to anything) and latest one 2023.

I tried to recursively delete the old directory, it shows windows like this:
![image](https://user-images.githubusercontent.com/46024870/217187309-f1b8faaf-fc4b-47e3-a4e7-f4ce1f1507e7.png)
I selected continues, there is no confirmation nor error, yet it isn't deleted. The only left on that directory is a directory `dev` and inside it is `null`.

How to delete it?

---

Update:
I managed to move the `dev/null` directory to parent root then delete the old MSYS2 dir. When I tried to delete the `null` file inside `dev` directory, an error occour.
![image](https://user-images.githubusercontent.com/46024870/217188136-720614cb-dd12-489b-8c71-af23a7fbc5c6.png)

---
Update 2:
I did some operations, rename directory, then moving, then rename to original directory name. I managed to put the `null` into latest msys2's `dev`, this is the result:
![image](https://user-images.githubusercontent.com/46024870/217188946-eed35b99-c93c-4474-b0b6-3a862f8d7e59.png)
I'm not similar with this file... it seems this is is not normal file.

MSYS2 is working normal and leftover are completely deleted. I think it's okay to get this discussion closed.

**Comment by rasa** [comment, marked as answer]
`del \\.\c:\path\to\nul` shoudl allow you to delete it.
